sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/api/Microsoft.Azure.WebPubSub.Common.netstandard2.0.cs

```diff
@@ -109,6 +109,7 @@ public partial class WebPubSubConnectionContext
         public System.Collections.Generic.IReadOnlyDictionary<string, object> States { get { throw null; } }
         [System.Text.Json.Serialization.JsonPropertyNameAttribute("userId")]
         public string UserId { get { throw null; } }
+        public bool TryGetState<T>(string stateKey, out T value) { throw null; }
```

Comment:
Question - We are adding this to provide a more efficient API to retrieve a state instead of getting all via the `States` property? Is it because the alternative is too cumbersome, or just not feasible in some cases?

Reply:
It's because the `object` type is not that easier to use. User requires to convert themselves each time. So provide method with T to let users to get the value directly.

Reply:
Ok. So, this is an enhancement. This can be part of the next update.